cluster/addons/cluster-monitoring/README.md

```diff
@@ -5,4 +5,16 @@ Heapster collects signals from kubelets and the api server, processes them, and
 
 More details can be found in [Monitoring user guide](http://kubernetes.io/docs/user-guide/monitoring/).
 
+## Troubleshooting
+
+Heapster supports up to 30 pods per cluster node. In clusters where there are more running pods, Heapster may be throttled or fail with OOM error. Starting with Kubernetes 1.9.2, Heapster resource requirements may be overwritten manually. [Learn more about Addon Resizer configuration](https://github.com/kubernetes/autoscaler/tree/master/addon-resizer#addon-resizer-configuration)
+
+### Important notices
+
+1. Decreasing resource requirements for cluster addons may cause system instability. The effects may include (but are not limited to):
+  - Metrics not being exported
```

Comment:
Add cluster versions that may be affected.

Reply:
All versions may be affected

Reply:
After a certain k8s version kubectl top no longer depends on Heapster, yet this page suggests it does.

Reply:
I wasn't aware that these changes are live, do you know when we switched? (If they're not live, this may change so I would leave this docs without promising anything)

Reply:
Well, https://github.com/kubernetes/kubernetes/pull/56206 was merged in January, so I think 1.10.0?